### PR TITLE
#17 ng add done 

### PR DESCRIPTION
**Pull request by nitishk72**
Hi
I had finished the **ng-add** see if everything is fine

### CODE REVIEW

**Comment by santoshyadav198613** [comment]
Thanks Nitish, will verify in evening. Thanks for PR.

File: command-builder/package.json [hunk 1]
```diff
@@ -1,19 +1,21 @@
 {
   "name": "@netlify-builder/deploy",
-  "version": "1.0.3",
-  "description": "",
-  "main": "command/index.js",
+  "version": "1.9.0",
```
Comment (santoshyadav198613) [inline command-builder/package.json]: Keep the version the same as previous one.

File: command-builder/ng-add/index.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import {
+    addPackageJsonDependency,
+    NodeDependency,
+    NodeDependencyType
+} from 'schematics-utilities';
+
+function addPackageJsonDependencies(): Rule {
+    return (host: Tree, context: SchematicContext) => {
+        const dependencies: NodeDependency[] = [
+            { type: NodeDependencyType.Default, version: '~1.9.0', name: 'netlify-builder/deploy' }
```
Comment (santoshyadav198613) [inline command-builder/ng-add/index.ts]: Here you can change the version to 1.0.3, will update the same on release.

File: command-builder/package.json (hunk 1 above)
Comment (nitishk72) [reply command-builder/package.json]: okay

File: command-builder/ng-add/index.ts (hunk 2 above)
Comment (nitishk72) [reply command-builder/ng-add/index.ts]: can you please edit the version and accept the PL.

Comment (santoshyadav198613) [reply command-builder/ng-add/index.ts]: cool

**Pull request merged**